common/test/acceptance/tests/video/test_studio_video_module.py

```diff
@@ -42,6 +45,38 @@ def setUp(self):
         )
 
         self.assets = []
+        self.addCleanup(YouTubeStubConfig.reset)
+
+    def _create_course_unit(self, youtube_stub_config=None, subtitles=False):
+        """
+        Create a Studio Video Course Unit and Navigate to it.
+
+        Arguments:
+            youtube_stub_config (dict)
+            subtitles (bool)
+
+        """
+        if youtube_stub_config:
+            YouTubeStubConfig.configure(youtube_stub_config)
+
+        if subtitles:
+            self.assets.append('subs_3_yD_cEKoCk.srt.sjson')
```

Comment:
nit: can this .sjson be renamed to something a newbie could understand? Like, while trolling through the test_root directory or some such?